src/ensembl/src/content/app/species-selector/state/speciesSelectorSelectors.ts

```diff
@@ -62,6 +63,16 @@ export const getCommittedSpecies = (state: RootState): CommittedItem[] => {
   return state.speciesSelector.committedItems;
 };
 
+export const getCommittedSpeciesById = (
+  state: RootState,
+  genomeId: string
+): CommittedItem | null => {
+  const allCommittedSpecies = getCommittedSpecies(state);
+  return (
+    find(allCommittedSpecies, ({ genome_id }) => genomeId === genome_id) || null
```

Comment:
Won't this be more readable if `genomeId === genome_id` is instead as `genomeId === species.genome_id`?

Reply:
Updated the syntax of this line.